### ISSUE TITLE
Correct the context menu item copy for managing containers

### ISSUE THREAD

**Issue by ryanfeeley** [mozilla/multi-account-containers#1278]
The contextual menu item for managing containers does not follow our guidelines for [title case](https://design.firefox.com/photon/copy/capitalization.html) or [ellipses](https://design.firefox.com/photon/copy/punctuation.html#ellipses).

<img width="203" alt="image" src="https://user-images.githubusercontent.com/68213/46220989-cdd9c580-c319-11e8-889a-031d443169cf.png">

**CURRENT**
Manage containers

**PROPOSED**
Manage Containers…

**Comment by ShivangiKakkar** [comment]
I would like to take this up and probably send the PR soon. :)

**Comment by groovecoder** [comment]
Wait, if you mean in the long-press context menu, I think that has to be done in the platform-level code - not in the add-on. @jonathanKingston ?

**Comment by ryanfeeley** [comment]
@groovecoder You are right, it would be platform level. Probably here: https://hg.mozilla.org/l10n/gecko-strings/file/default/browser/chrome/browser/browser.properties#l899

**Comment by jonathanKingston** [comment]
@ShivangiKakkar @ryanfeeley is correct in where he has linked to. This will need a bug filing on Bugzilla instead... https://bugzilla.mozilla.org/show_bug.cgi?id=1450973 which Ryan also raised there too.

In [another bug](https://github.com/mozilla/multi-account-containers/issues/945#issuecomment-425278140) I explained how this is possible to find and get going with changing this code. I will be available to work through anything you need.

**Comment by grahamperrin** [comment]
At a glance, this appears to be fixed.
